launch-checklist.md

```diff
@@ -1 +1,21 @@
+# Replicache Launch Checklist
 
+Before you launch with Replicache in your product, it's a good idea to double-check that you have correctly covered all the small details of integration. This list can help you determine if you might have missed a detail, or deferred and then forgotten about one.
+
+- **JS SDK**
+  - if you wish to change the type of a mutator (eg, the number or type of its arguments) you must choose a new name; Replicache does not handle mutator versioning
+  - if you wish to version the schema of the mutations in `Push` and the `clientView` returned in `Pull`, version information is included in `ReplicacheOptions.schemaVersion`
```

Comment:
```suggestion
  - if you wish to version the schema of the mutations in **Push** and the **clientView** returned in **Pull**, version information is included in `ReplicacheOptions.schemaVersion`
```

Push, clientView and Pull are not methods. We do have a `pull` method but that is it.

Reply:
ok, i was using code style for everything that is a technical entity, believe now code style is only for actual things we have in the code.